src/index.ts

```diff
@@ -0,0 +1,227 @@
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+const { argv } = yargs(hideBin(process.argv));
+
+import { Locations } from "./locations";
+import { logger } from "./logger";
+import { readConfig } from "./readConfig";
+// eslint-disable-next-line unused-imports/no-unused-imports
+import { server } from "./server";
```

Comment:
We can simply import and skip the from here
```suggestion
import "./server";
```